UPGRADE-4.x.md

```diff
@@ -1,2 +1,7 @@
 UPGRADE 4.x
 ===========
+
+### Deprecated UserGenderListType
+
+Relying on this class to provide user gender list on sonata forms is deprecated and
+will be removed in 5.0. You should use choice type instead.
```

Comment:
Either "the choice type" or `ChoiceType` (or even `Symfony\Component\Form\Extension\Core\Type\ChoiceType`)

Reply:
Fixed